### ISSUE TITLE
Files are not having extension

### ISSUE THREAD

**Issue by ashishrajsrivastava** [AzureTalk/Script#2]
It seems most of the files are powershell files but extension is hidden for some reason. can we have the files updated with extension names , will help to determine if some artifacts got uploaded instead of source file.

**Comment by DexterPOSH** [comment]
@ashishrajsrivastava I can work on this next week.

**Comment by DexterPOSH** [comment]
Also using Readthedocs.org to generate dynamic content out of those MD files is always an option.
What say? 
I can work on this 😄

**Comment by ashishrajsrivastava** [comment]
Going through readthedocs.com, this is something new for me 🤓

**Comment by DexterPOSH** [comment]
Ah! cool. The idea is to use Markdown to generate our wiki content dynamically and serve it over a URL 😎 
I will open another issue to work on it, once the markdown conversion is complete.

**Comment by DexterPOSH** [comment]
As per the PR https://github.com/KloudEzy/AzureDevOps/pull/4 this should be fixed now.
Now time to work on Readthedocs integration.

**Comment by ashishrajsrivastava** [comment]
files has been updated with md extension as part of #4
